response_operations_ui/controllers/uaa_controller.py

```diff
@@ -136,3 +135,43 @@ def change_user_password(email, password):
 
     return change_password(access_token=access_token, user_code=password_reset_code,
                            new_password=password)
+
+
+def create_user_account(email, password, user_name, first_name, last_name):
+    access_token = login_admin()
+
+    headers = {'Content-Type': 'application/json',
+               'Accept': 'application/json',
+               'Authorization': f'Bearer {access_token}'}
+
+    payload = {
+        "userName": user_name,
+        "name": {
+            "formatted": f"{first_name} {last_name}",
+            "givenName": first_name,
+            "familyName": last_name
+        },
+        "emails": [{
+            "value": email,
+            "primary": True
+        }],
+        "active": True,
+        "verified": True,
+        "password": password
+    }
+
+    url = f"{app.config['UAA_SERVICE_URL']}/Users"
+    response = requests.post(url, data=dumps(payload), headers=headers)
+    try:
+        response.raise_for_status()
+        return
+    except HTTPError:
+        if response.status_code == 409:
+            # Username already exists
+            errors = {'user_name': ["Username already in use; please choose another"]}
+        else:
+            errors = {'status_code': response.status_code, 'message': response.reason}
+            logger.error('Received an error when creating an account in UAA',
```

Comment:
You should add the reason to the logger.error here.  The way the code currently works is the reason only ever makes it to the user (and never the person looking at the logs) if the error has (user_name) in it.  Adding this will aid in debugging